ui/components/src/GaugeChart/GaugeChart.tsx

```diff
@@ -53,7 +53,7 @@ export function GaugeChart(props: GaugeChartProps) {
         {
           type: 'gauge',
           center: ['50%', '65%'],
-          radius: '100%',
+          radius: '86%',
```

Comment:
Setting this to a value that maintains the ratio that existed previously. Let me know if you think that should be tweaked.